t.py

```diff
@@ -0,0 +1,5 @@
+import json
+import os
+def whitespace_issues():
+    x = 'should be double quotes'
```

Comment:
F841: local variable 'x' is assigned to but never used  <!-- Automatically posted by Lintly -->